charter.md

```diff
@@ -0,0 +1,41 @@
+# Interop Team Charter
+
+The Interop Team aims to make the web more interoperable in key areas, prioritized by web developer and user needs. This team is part of [the web-platform-tests project](https://github.com/web-platform-tests/wpt), the main test suite for the web platform.
+
+The team is responsible for maintaining [Compat 2021](https://wpt.fyi/compat2021), [Interop 2022](https://wpt.fyi/interop-2022), and defining future interoperability efforts and metrics.
```

Comment:
(if my other suggestions are accepted) remove this, as it will be covered under "scope".

Reply:
I kept this wording (with links) but put it in the scope instead.